src/player.cpp

```diff
@@ -8200,7 +8204,7 @@ int player::item_store_cost( const item& it, const item& /* container */, bool p
 
 int player::item_reload_cost( const item& it, const item& ammo, long qty ) const
 {
-    if( ammo.is_ammo() ) {
+    if( ammo.is_ammo() || ammo.has_flag( "SPEEDLOADER" ) ) {
         qty = std::max( std::min( ammo.charges, qty ), 1L );
```

Comment:
This looks wrong: this makes speedloader reload time scale with amount of ammo inserted into it.

Reply:
Yep, I'll back this out.